sdk/cosmos/azure-cosmos-benchmark/src/main/java/com/azure/cosmos/benchmark/Main.java

```diff
@@ -73,6 +74,8 @@ public static void main(String[] args) throws Exception {
             System.err.println("INVALID Usage: " + e.getMessage());
             System.err.println("Try '-help' for more information.");
             throw e;
+        } finally {
+            System.exit(0);
```

Comment:
why do we need this?
This will cause problem for tests directly invoking Main:main()